docs/examples/bearing_only_example.py

```diff
@@ -0,0 +1,267 @@
+# 4) Run the simulation and create the plots
+
+# some general imports
+import numpy as np
+from matplotlib import pyplot as plt
+
+from datetime import datetime
+from datetime import timedelta
+
+# Load Stone Soup materials
+from stonesoup.types.state import State, GaussianState
+from stonesoup.types.array import StateVector, CovarianceMatrix
+from stonesoup.models.transition.linear import (CombinedLinearGaussianTransitionModel, ConstantVelocity)
+from stonesoup.models.measurement.nonlinear import Cartesian2DToBearing
+
+# Load the filter components
+from stonesoup.updater.kalman import ExtendedKalmanUpdater
+from stonesoup.predictor.kalman import ExtendedKalmanPredictor
+from stonesoup.deleter.time import UpdateTimeStepsDeleter
+from stonesoup.tracker.simple import SingleTargetTracker
+
+# set a random seed and start of the simulation
+np.random.seed(2001)
+start_time = datetime.now()
+
+# %%
+# 1) Create the moving platform and the Bearing-Only radar
+# --------------------------------------------------------
+# Firstly, we create the initial state of the platform, including the origin point and the
+# cartesian (x,y) movement direction. Then, we create a transition model (in 2D cartesian coordinates)
+# of the platform.
+# At this point, we can setup the Radar which receives only the bearing measurements from the targets using the
+# :class:`~.RadarBearing` object.
+
+# Import the platform to place the sensor
+from stonesoup.platform.base import MovingPlatform
+
+# Define the platform location, place it in the origin, and define its cartesian movements.
+# In addition specify the position and velocity mapping. This is done in 2D cartesian coordinates.
+
+platform_state_vector = StateVector([[0], [-5], [0], [-7]])
+position_mapping = (0, 2)
+velocity_mapping = (1, 3)
+
+# Create the initial state (position and time)
+platform_state = State(platform_state_vector, start_time)
+
+# Create a platform transition model, let's assume it is moving in a straight line
+# from its starting place
+platform_transition_model = CombinedLinearGaussianTransitionModel([
+    ConstantVelocity(0.0), ConstantVelocity(0.0)])
+
+# We can instantiate the platform initial state, position and velocity mapping and 
+# the transition model
+platform = MovingPlatform(states=platform_state,
+                          position_mapping=position_mapping,
+                          velocity_mapping=velocity_mapping,
+                          transition_model=platform_transition_model)
+
+# At this stage, we need to create the sensor, let's import the RadarBearing. 
+# This sensor only provides the bearing measurements from the target detections, 
+# the range is not specified.
+from stonesoup.sensor.radar.radar import RadarBearing
+
+# Configure the radar noise, since we are using just a dimension we need to specify only the
+# noise associated with the bearing, we assume a bearing accuracy of +/- 0.025 degrees for 
+# each measurement
```

Comment:
```suggestion
# Configure the radar noise, since we are using just a single dimension we need to specify only the
# noise associated with the bearing dimension, we assume a bearing accuracy of +/- 0.025 degrees for 
# each measurement
```